### ISSUE TITLE
SUGGESTION: Lowering the transfer time from five and a half to four hours.

### ISSUE THREAD

**Issue by MrSign** [VOREStation/VOREStation#4086]
#### Brief description of the issue
Rounds are just too long to keep things enjoyable. People get restless by the four hour mark and it is quite the slog to wait for an extra hour and a half for a CHANCE to get a transfer.

#### What you expected to happen
Four hours of FUN!

#### What actually happened
Four hours of fun then one to two hours of going 'PLEEEASE TRANSFER!'

#### Steps to reproduce
-Play a round of Vorestation from start to finish.
-Record the time where you get bored and there's nothing to do because EVERYTHING POSSIBLE has been done.

#### Code Revision
-6831216aa58b34b2d6610f7de92c556b69272fbc 
 
#### Anything else you may wish to add:
Basically, just have the transfer vote at four hours in. If people still wanna play, extend it to like the five/five and a half hour mark. Ty ty

**Comment by SpadesNeil** [comment]
More elaborate no:

Any shorter, and we'd not be able to do the RP scenes _for vore_, which is the intended purpose of the station. Most scenes take 2-3 hours. 5 hours is already a reduction from 6 hours, which was a reduction from 8 hours, which was the original shift length.

I'm not lowering it any further, or the length of a shift becomes less than twice the length of one detailed scene, which thereby deters players from joining until next shift, by which point many people just don't join at all.

**Comment by TheDavestDave** [comment]
I think compers has something to add here. Because it has a vote every 2 hours to restart and they always get to the max 6 hour round. While most people vote for the round to reset at our 5 hour mark. 

A good example of why a staggered voteing sytem would be good if there is three people on ath the 2 hour mark. And now everyone has to wait 3 hours to get a scene.

**Comment by kcin2001** [comment]
the sec not doing lewds is because most of the people who get put in the brig are total shitheads @TheDavestDave
